src/main/resources/static/js/information.js

```diff
@@ -1,25 +1,25 @@
+/* global getFrontEndServiceUrl backendEndpoints */
 jQuery(document).ready(function () {
 
     // Fetch injected URL from DOM
-    var frontEndServiceUrl = $('#frontendServiceUrl').text();
-    var frontEndVersion = $('#frontEndVersion').text();
-    var frontEndApplicationPropertiesVersion = $('#frontEndApplicationPropertiesVersion').text();
-    var frontEndAppName = $('#frontendAppName').text();
+    var frontEndVersion = $("#frontEndVersion").text();
+    var frontEndApplicationPropertiesVersion = $("#frontEndApplicationPropertiesVersion").text();
+    var frontEndAppName = $("#frontendAppName").text();
 
-    var body = document.getElementById('eiPageFrame');
+    var body = document.getElementById("eiPageFrame");
     var generalEIInfoLabel = "General Eiffel Intelligence Information";
     var generalEIFrontEndInfoLabel = "General Eiffel Intelligence Front-End Information";
 
     function createTable() {
-        var tbl = document.createElement('table');
-        tbl.setAttribute('class', 'table table-bordered table-striped dataTable table-text-setting');
+        var tbl = document.createElement("table");
+        tbl.setAttribute("class", "table table-bordered table-striped dataTable table-text-setting");
         return tbl;
     }
 
     function createLabel(inputText) {
-        var label = document.createElement('p');
+        var label = document.createElement("p");
```

Comment:
since this is a character we can keep the old code ?

Reply:
Will read up on best practice.

Reply:
Doesn't Codacy usually complain about single quote?